recipes/irida-linker/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: irida-linker
+  version: "1.0.1"
+
+source:
+  fn: 1.0.1.tar.gz
+  url: https://github.com/phac-nml/irida-linker/archive/1.0.1.tar.gz
+  md5: 958689540fb5fde2746af9354438d695
+  
+build:
+  number: 0
+  skip: True  # [osx]
+  
+requirements:
+  build:
+    - perl-lwp-simple
+    - perl-json
+    - perl-getopt-long
+    - perl-file-path
+    - perl-config-simple
+    - perl-termreadkey
+    - perl-http-message
+    - perl-libwww-perl
+    
+  run:
+    - perl-lwp-simple
+    - perl-json
+    - perl-getopt-long
+    - perl-file-path
+    - perl-config-simple
+    - perl-termreadkey
+    - perl-http-message
+    - perl-libwww-perl
+test:
+  commands:
+    - ngsArchiveLinker.pl
+extra:
+  skip-lints:
```

Comment:
You can skip these skips, you follow our best practices very much! :)

Reply:
Removed.